cluster/gce/config-common.sh

```diff
@@ -84,6 +84,20 @@ function get-node-ip-range {
   echo "${suggested_range}"
 }
 
+function get-cluster-ip-range {
```

Comment:
Unfortunately no - because this is used by the cluster startup scripts which have no connection to test_context (a purely e2e-testing framework concept). However, I made sure that the defaults do match (for non-default ones, we can set the env var in kubetest which still lets us clear away one more env var from job configs :)

On a related note - I personally prefer that more and more of such variables should be auto-calculated by the startup scripts as otherwise it becomes more error-prone when someone (who probably didn't notice it) has to do it.